tests/e2e/pages/contact-view.page.ts

```diff
@@ -0,0 +1,16 @@
+import { BasePage } from './base.page';
+
+/**
+ * Contact view Page
+ */
+export class ContactView extends BasePage {
```

Comment:
I am not sure yet if this would be a good idea to discuss this now, but since the number of pages will be growing once we add more tests and pages. Shouldn't we group the pages based on civicrm objects or urls for example,

tests/e2e/pages/contact/contact.view.page.ts
tests/e2e/pages/contact/contact/individual-create.page.ts  

instead of adding all the pages inside `pages `directory.

Reply:
I think it is fine for time being, maybe in the next test we are writing we can move things around.